spec/models/crime_application_spec.rb

```diff
@@ -139,4 +139,85 @@
       end
     end
   end
+
+  describe 'court hearing name' do
```

Comment:
I like this Mo I think it is pretty clear, to iterate over this rather than be completely explicit.